### ISSUE TITLE
expand buttons would be better if the said what they were expanding in the text

### ISSUE THREAD

**Issue by jnurthen** [stevefaulkner/HTML5accessibility#31]
The word expand is added for each button. While not a WCAG failure as there is context in the table cell, it would be much better if this text actually included what they were expanding.

**Comment by dstorey** [comment]
See https://github.com/stevefaulkner/HTML5accessibility/issues/26#issuecomment-212983061
